cmd/skopeo/list_tags.go

```diff
@@ -28,6 +29,20 @@ type tagsOptions struct {
 	retryOpts *retry.RetryOptions
 }
 
+var transportHandlers = map[string]func(ctx context.Context, sys *types.SystemContext, opts *tagsOptions, userInput string) (repositoryName string, tagListing []string, err error){
+	docker.Transport.Name():  listDockerRepoTags,
+	archive.Transport.Name(): listDockerArchiveTags,
+}
+
+// getSupportTransport return all the supported transports
+func getSupportTransport(joinStr string) string {
+	supportTranspors := make([]string, 0, len(transportHandlers))
+	for handlerName := range transportHandlers {
+		supportTranspors = append(supportTranspors, handlerName)
+	}
+	return strings.Join(supportTranspors, joinStr)
```

Comment:
`sort.Strings(supportedTransports)`, otherwise the iteration order of the hash is not guaranteed to be consistent over time.